crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -88,6 +100,10 @@ def database
 
         true
       end
+
+      def is_proposal?(filename)
```

Comment:
Style/PredicateName: Rename `is_proposal?` to `proposal?`. (https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)